test/distro/diagram-js-minimap.js

```diff
@@ -0,0 +1,24 @@
+const {
+  expect
+} = require('chai');
+
+describe('diagram-js-minimap', function() {
+
+  it('should expose CJS bundle', function() {
+
+    const {
+      DiagramJSMinimap
+    } = require('../../dist/diagram-minimap.umd.js');
```

Comment:
here we have to use `require('../..')` because we want to use the cjs bundle (`dist/index.js`, as pointed out in the `package.json`)